### ISSUE TITLE
Focused Launch: Try switching styles to CSS-in-JS in packages

### ISSUE THREAD

**Issue by ciampo** [Automattic/wp-calypso#47112]
As discussed in https://github.com/Automattic/wp-calypso/pull/47062#discussion_r517477321, we can start using CSS-is-JS in packages following the pattern from composite-checkout.

**Comment by Robertght** [comment]
@ciampo Is this still relevant? If not, I noticed more tasks are stale as well:

https://github.com/Automattic/wp-calypso/issues?q=is%3Aissue+is%3Aopen+sort%3Aupdated-asc+label%3A%22Focused+Launch%22+label%3A%22%5BStatus%5D+Stale%22

**Comment by ciampo** [comment]
I don't think so, but @razvanpapadopol may be able to give a more informed answer

**Comment by razvanpapadopol** [comment]
@Robertght, some of the tasks may be still relevant but the whole project is on pause for now. I've added the "Hold" label and an explanatory card on the ["Now" column of the project board](https://github.com/Automattic/wp-calypso/projects/103#column-11984049).
